### PR TITLE
Update README.md with information about docker

### PR DESCRIPTION
**Pull request by violet4**
point to spiffworkflow.org docker guide and mention the differences between docker-compose.yml and editor.docker-compose.yml

i ran `diff -u <(curl -s https://raw.githubusercontent.com/sartography/spiff-arena/main/docker-compose.yml) <(curl -s https://raw.githubusercontent.com/sartography/spiff-arena/main/editor.docker-compose.yml)` and got this, which i based my documentation update and commit message on:

```diff
--- /dev/fd/63	2023-08-29 15:22:57.793609978 -0700
+++ /dev/fd/62	2023-08-29 15:22:57.793609978 -0700
@@ -1,7 +1,7 @@
 services:
   spiffworkflow-frontend:
     container_name: spiffworkflow-frontend
-    image: ghcr.io/sartography/spiffworkflow-frontend:latest
+    image: ghcr.io/sartography/spiffworkflow-frontend:main-latest
     depends_on:
       spiffworkflow-backend:
         condition: service_healthy
@@ -13,7 +13,7 @@
 
   spiffworkflow-backend:
     container_name: spiffworkflow-backend
-    image: ghcr.io/sartography/spiffworkflow-backend:latest
+    image: ghcr.io/sartography/spiffworkflow-backend:main-latest
     environment:
       SPIFFWORKFLOW_BACKEND_APPLICATION_ROOT: "/"
       SPIFFWORKFLOW_BACKEND_ENV: "local_development"
@@ -25,21 +25,21 @@
       SPIFFWORKFLOW_BACKEND_BPMN_SPEC_ABSOLUTE_DIR: "/app/process_models"
       SPIFFWORKFLOW_BACKEND_CONNECTOR_PROXY_URL: "http://spiffworkflow-connector:8004"
       SPIFFWORKFLOW_BACKEND_DATABASE_TYPE: "sqlite"
-      SPIFFWORKFLOW_BACKEND_DATABASE_URI: "sqlite:////app/db_volume/db.sqlite3"
       SPIFFWORKFLOW_BACKEND_LOAD_FIXTURE_DATA: "false"
+      SPIFFWORKFLOW_BACKEND_AUTHENTICATION_DISABLED: "true"
       SPIFFWORKFLOW_BACKEND_OPEN_ID_CLIENT_ID: "spiffworkflow-backend"
       SPIFFWORKFLOW_BACKEND_OPEN_ID_CLIENT_SECRET_KEY: "my_open_id_secret_key"
       SPIFFWORKFLOW_BACKEND_OPEN_ID_SERVER_URL: "http://localhost:${SPIFF_BACKEND_PORT:-8000}/openid"
       SPIFFWORKFLOW_BACKEND_PERMISSIONS_FILE_NAME: "example.yml"
       SPIFFWORKFLOW_BACKEND_PORT: "${SPIFF_BACKEND_PORT:-8000}"
-      SPIFFWORKFLOW_BACKEND_RUN_BACKGROUND_SCHEDULER: "true"
+      SPIFFWORKFLOW_BACKEND_RUN_BACKGROUND_SCHEDULER: "false"
       SPIFFWORKFLOW_BACKEND_UPGRADE_DB: "true"
       SPIFFWORKFLOW_BACKEND_URL_FOR_FRONTEND: "http://localhost:${SPIFFWORKFLOW_FRONTEND_PORT:-8001}"
     ports:
       - "${SPIFF_BACKEND_PORT:-8000}:${SPIFF_BACKEND_PORT:-8000}/tcp"
     volumes:
-      - ./process_models:/app/process_models
-      - spiffworkflow_backend_db:/app/db_volume
+      - "${SPIFF_EDITOR_BPMN_SPEC_DIR:-./process_models}:/app/process_models"
+      - ./log:/app/log
     healthcheck:
       test: "curl localhost:${SPIFF_BACKEND_PORT:-8000}/v1.0/status --fail"
       interval: 10s
@@ -62,7 +62,6 @@
       timeout: 5s
       retries: 20
 
-
 volumes:
-  spiffworkflow_backend_db:
+  spiffworkflow_backend:
     driver: local
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,12 @@ Get the app running so you can access the frontend at http://localhost:7001 in y
 
     ./bin/run_cypress_tests_locally
 
+## Docker
+
+For full instructions, see [Running SpiffWorkflow Locally with Docker](https://www.spiffworkflow.org/posts/articles/get_started_docker/).
+
+The docker-compose.yml file is for running an instance of spiff-arena while editor.docker-compose.yml is for testing the spiff-arena codebase.
```
Comment (jbirddog) [inline README.md:40]: Appreciate adding this section to the README. There isn't a great way to know this, which your contribution will be correcting, but the editor flavor was added to serve the use case of wanting to edit bpmn diagrams that were intended to be executed outside of spiff arena. There were some apps/libraries that used spiffworkflow directly but had no editor. Anyway if that last part about the editor could focus more on that aspect i'd say this is good to go.

Comment (violet4) [reply README.md:40]: thank you so much for your response! i'm not 100% certain i understand, so i'll try to capture what you said and rephrase for the docs, and please let me know if it sounds okay.

* The docker-compose.yml file is for running a full-fledged instance of spiff-arena while editor.docker-compose.yml provides BPMN graphical editor capability to libraries and projects that depend on SpiffWorkflow but have no built-in BPMN edit capabilities.

i would love to add more detail/nuance to this description but don't know enough to elaborate with confidence, so i'll pose a question to ensure my understanding: would the docker-compose of another project include editor.docker-compose.yml in its own compose file and allow that project to defer edits to the services running inside the editor container, like an "open external editor" button/feature?

thanks again!
~violet

Comment (jbirddog) [reply README.md:40]: That bullet point sums it up nicely. What you are describing with the compose file is very close to how I used the editor feature for the command line app I was working on. There I had a Makefile that had a [run-editor](https://github.com/jbirddog/git-debranch/blob/main/Makefile#L18) target which would kick off a [script](https://github.com/sartography/spiff-arena/blob/main/bin/run_editor#L23) and open spiff arena in editor mode pointing to the process models for my app.

Comment (violet4) [reply README.md:40]: i didn't forget about this, i'll get back to it tomorrow, but for now, time to sleep..

Comment (violet4) [reply README.md]: updated based on our recent discussion of `editor.docker-compose.yml` 👍️🤞 hope it passes muster now :)

**Review by jbirddog — APPROVED** [review]

**Comment by burnettk** [comment]
thank you very much for contributing, @violet4 🙇

**Pull request merged**